src/Messages/Interactions/TSIncomingMessage.h

```diff
@@ -120,6 +104,7 @@ extern NSString *const TSIncomingMessageWasReadOnThisDeviceNotification;
 + (nullable instancetype)findMessageWithAuthorId:(NSString *)authorId timestamp:(uint64_t)timestamp;
 
 @property (nonatomic, readonly) NSString *authorId;
+@property (nonatomic, readonly) UInt32 sourceDeviceId;
```

Comment:
And it's okay that older messages in the data store will have a source device id of zero?